ratatui-core/src/backend/test.rs

```diff
@@ -233,10 +232,12 @@ impl fmt::Display for TestBackend {
     }
 }
 
+type InfallibleResult<T> = core::result::Result<T, core::convert::Infallible>;
```

Comment:
A more common way of writing this might be something like:
```suggestion
type Result<T, E = core::convert::Infallible> = core::result::Result<T, E>;
```